### DISCUSSION TITLE
Having issues with CMake trying to do a custom build

### DISCUSSION THREAD

**Discussion by paul-fornage** [iNavFlight/inav#7260] [Q&A] [answered]
I made [this](https://stackoverflow.com/questions/68368032/cmake-error-problem-with-archive-write-finish-entry-write-to-restore-size-fa) post on Stack Overflow to search for help, but I figured since I was following [this document](https://github.com/iNavFlight/inav/blob/master/docs/development/Building%20in%20Windows%2010%20or%2011%20with%20Linux%20Subsystem.md), maybe someone here knows how to fix this or can tell me where to unzip the file manually (if that is even possible). The error is "_CMake Error: Problem with archive_write_finish_entry(): Write to restore size failed_" and while many similar problems can be found online, I have never seen that exact error message, which is weird considering it happened on two different computers. Don't rule anything out as too obvious, I'm that dumb.

**Comment by CandiceFPV** [comment]
Let me know if you get this figured out, I am having the same issue.

**Reply by paul-fornage** [reply]
I just deleted inav from my c drive and started over again, that worked

**Comment by paul-fornage** [comment, marked as answer]
I just deleted inav from my c drive and started over again, that worked
